docs/tutorials/1_rosbag.md

```diff
@@ -0,0 +1,64 @@
+# Tutorial 1: RViz basics and replaying a rosbag
+The purpose of this exercise is to gain familiarity with the basic functionality of RViz and how to replay a rosbag. [RVIz](http://wiki.ros.org/rviz) is a 3D visualization tool for the ROS middleware that Autoware is based on, and [rosbag](http://wiki.ros.org/rosbag) is the file format used to record and playback data from a ROS-based system.
+
+Before starting, please download and unpack the following files:
+* [Tutorial 1 map files](https://drive.google.com/open?id=1ovrJcFS5CZ2H51D8xVWNtEvj_oiXW-zk)
+* [Tutorial 1 rosbag](https://drive.google.com/open?id=1BFcNjIBUVKwupPByATYczv2X4qZtdAeD)
+
+> Note that these are the same files used in the [rosbag quick launch exercise](../../README.md#rosbag-simulation)
+
+## Launch RViz, start and pause rosbag playback
+1. Open two terminal windows and run the following commands in *both* terminals
+```
+cd ~/workspace/AutowareArchitectureProposal
```

Comment:
`~/workspace/AutowareArchitectureProposal` is `~/workspace/AutowareArchitectureProposal.proj` now.
This change should be reflected to all command examples in this repository.

Reply:
@UMiho 470b6e0ac675506e1575e5b6f98dc0f2a3be5b80 fixes the issue!